tests/unit/amo/components/TestHeader.js

```diff
@@ -40,4 +63,95 @@ describe(__filename, () => {
     expect(root.find('.Header-title').prop('children'))
       .toContain('Firefox Add-ons');
   });
+
+  it('displays `login` text when user is not connected', () => {
+    const { store } = dispatchClientMetadata();
+    const fakeI18n = getFakeI18nInst();
+    const allProps = { i18n: fakeI18n, location: {}, query: '' };
+
+    const wrapper = mount(
```

Comment:
Is it not possible to just pass `store` and `i18n` directly to `Header`? Or will that cause issues with child components that get rendered?